### PR TITLE
Make dev-console project selector sticky again

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes https://issues.redhat.com/browse/ODC-2770

/kind bug

### CODE REVIEW

File: frontend/packages/dev-console/src/components/MetricsPage.tsx [hunk 1]
```diff
@@ -16,8 +16,7 @@ export interface MetricsPageProps {
 const MetricsPage: React.FC<MetricsPageProps> = ({ match }) => {
   const namespace = match.params.ns;
   return (
-    <>
-      <NamespaceBar />
+    <NamespacedPage variant={NamespacedPageVariants.light}>
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/MetricsPage.tsx]: FWIW, this will become a conflict once #4053 merges. It's removing this page.

**Comment by karthikjeeyar** [comment]
/retitle Bug 1808432: Make dev-console project selector sticky again

**Pull request merged**